templates/workspace_services/gitea/porter.yaml

```diff
@@ -72,6 +72,10 @@ parameters:
   - name: key_store_id
     type: string
     default: ""
+  - name: is_exposed_externally
```

Comment:
Do we really need this option?
This service wasn't exposed publicly in the past and the source of the issue is similar to this https://github.com/microsoft/AzureTRE/issues/4199 which is probably due to a change in Azure and/or AzureRM provider (many months ago).

Reply:
IMO we shouldn't allow the option of exposing it externally. @marrobi do you agree?

Reply:
It existed in template_schema.json so appears in the UI, not sure that it hurts to offer the option?  But needs resolving one way or the other.

Reply:
@tamirkamara looking better?